React/Modules/RCTUIManagerUtils.m

```diff
@@ -98,6 +99,6 @@ void RCTUnsafeExecuteOnUIManagerQueueSync(dispatch_block_t block)
 NSNumber *RCTAllocateRootViewTag()
 {
   // Numbering of these tags goes from 1, 11, 21, 31, ..., 100501, ...
-  static int64_t rootViewTagCounter = -1;
-  return @(OSAtomicIncrement64(&rootViewTagCounter) * 10 + 1);
+  static _Atomic int64_t rootViewTagCounter = 0;
```

Comment:
From [the docs](https://en.cppreference.com/w/c/atomic/atomic_fetch_add):

> Atomically replaces the value pointed by obj with the result of addition of arg to the old value of obj, and **returns the value obj held previously**.

So that’s why this should initialise with `0` instead of `-1`, like previously was the case.

Do you know if there are tests that cover this API?

Reply:
I don't know of any tests per se, but the RNTester app crashed immediately when this was initialized to `-1`.